core/src/test/java/org/frankframework/pipes/EscapePipeTest.java

```diff
@@ -51,6 +51,18 @@ public void testPartialEscapedMultiLineXmlInput() throws Exception { //double es
 		TestAssertions.assertEqualsIgnoreCRLF(expected, result.getResult().asString());
 	}
 
+	@Test
+	public void testPartialEscapedMultiLineXmlInputMultiOccurences() throws Exception {
+		pipe.setSubstringStart("<message>");
+		pipe.setSubstringEnd("</message>");
+		configureAndStartPipe();
+
+		PipeRunResult result = doPipe(getResource("multi-line-partial-escaped-multi-occurences.xml"));
+		String expected = getResource("multi-line-partial-escaped-multi-occurences.result").asString();
+		System.out.println(result.getResult().asString());
+		TestAssertions.assertEqualsIgnoreCRLF(expected, result.getResult().asString());
+	}
+
```

Comment:
Can you also add a test case (or change the current one), to proof it does not change the parts outside the start/end Substrings? At least, that is how I think it should work.

Reply:
It does do that right?
It compares a new xml file where only the content between the message tags is being replaced

Reply:
Yes, but perhaps you can also proof by (the same?) a test that outside the start/end substrings it does not replace chars.